src/qt/sendcoinsdialog.cpp

```diff
@@ -274,6 +276,71 @@ void SendCoinsDialog::on_sendButton_clicked()
         return;
     }
 
+    if (have_warning) {
+        struct prior_usage_info_t {
+            CAmount total_amount{0};
+            int num_txs{0};
+            qint64 tx_time_oldest;
+            qint64 tx_time_newest;
+        };
+        QMap<QString, prior_usage_info_t> prior_usage_info;
+        {
+            QStringList addresses;
+            for (const auto& recipient : recipients) {
+#ifdef ENABLE_BIP70
+                if (recipient.paymentRequest.IsInitialized()) continue;
```

Comment:
Maybe add a comment here. Why not put `ifndef ENABLE_BIP70` before the entire `{` block? Can users manually add destinations to a BIP70 payment?

Reply:
> Can users manually add destinations to a BIP70 payment?

BIP70 is no longer supported. Looks like this needs to be rebased on master and any `ENABLE_BIP70` additions needs to be removed.